murakami/runners/ooniprobe.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import shutil
+import subprocess
+import json
+import datetime
+
+from murakami.errors import RunnerError
+from murakami.runner import MurakamiRunner
+
+logger = logging.getLogger(__name__)
+
+
+class OONIProbeClient(MurakamiRunner):
+    """Run a test with ooniprobe."""
+    def __init__(self, config=None, data_cb=None,
+        location=None, network_type=None, connection_type=None,
+        device_id=None):
+        super().__init__(
+            title="ooniprobe",
+            description="Test the blocking of websites and apps.",
+            config=config,
+            data_cb=data_cb,
+            location=location,
+            network_type=network_type,
+            connection_type=connection_type,
+            device_id=device_id
+        )
+
+    def _start_test(self):
+        logger.info("Starting ooniprobe test...")
+        if shutil.which("ooniprobe") is not None:
+            output = None
+            
+            # Empty the ooniprobe database.
```

Comment:
```suggestion
            # Empty the ooniprobe database.
            # We do that to avoid wasting disk space with ooniprobe's database (which we don't use).
```